docs/source/building/environment-vars.rst

```diff
@@ -37,8 +37,8 @@ override this behavior.)
     - Version of the package being built
   * - ``PKG_BUILDNUM``
     - Build number of the package being built
-  * - ``PATH``
-    - Prepended by the build prefix bin directory
+  * - ``MAKEFLAGS``
+    - Inherited from your shell environment.
```

Comment:
Suggestion for addition:

May be used to set additional arguments to make, such as -j2, which will use 2 CPU cores to build your recipe.